funnel/views/session.py

```diff
@@ -93,15 +93,15 @@ def view(self):
         return dict(project=self.obj.project, active_session=session_data(self.obj, with_modal_url='view_popup'),
             from_date=date_js(self.obj.project.date), to_date=date_js(self.obj.project.date_upto),
             sessions=session_list_data(self.obj.project.scheduled_sessions, with_modal_url='view_popup'),
-            timezone=timezone(self.obj.project.timezone).utcoffset(datetime.now()).total_seconds(),
+            timezone=self.obj.project.timezone.utcoffset(datetime.utcnow()).total_seconds(),
```

Comment:
This parameter should be `tzoffset`, not `timezone`. @vidya-ram this will need changes in the JS where the name is used.